analyzers/src/Directory.Build.targets

```diff
@@ -4,7 +4,8 @@
 
   <PropertyGroup>
     <!-- `BinariesFolder` - binary files folder used by ITs and java packaging -->
-    <BinariesFolder>$(MSBuildThisFileDirectory)\..\packaging\binaries\</BinariesFolder>
+    <BinariesFolder>$(MSBuildThisFileDirectory)..\packaging\binaries\</BinariesFolder>
+    <InternalBinariesFolder>$(BinariesFolder)internal\</InternalBinariesFolder>
```

Comment:
What about `BinariesFolderInternal`? It follows also the value

Reply:
It makes more sense indeed.